page-controller/README.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: pattern
+title: Page Controller
+folder: parameter-controller
+permalink: /patterns/parameter-object/
+categories: Structural
+language: en
+tags:
+- Decoupling
+---
+## Intent
+It is an approach of one page leading to one logical file that handles actions or requests on a website.
+
```

Comment:
The explanation section is missing, see https://github.com/iluwatar/java-design-patterns/wiki/02.-Pattern-template. Check out the other patterns how it needs to be written.